Livecheckables/apache-brooklyn-cli.rb

```diff
@@ -0,0 +1,4 @@
+class ApacheBrooklynCli
+  livecheck :url   => "https://github.com/apache/brooklyn-client.git",
+            :regex => %r{^rel\/apache-brooklyn-?(\d+(?:\.\d+)+)$}i
```

Comment:
```suggestion
            :regex => %r{^(?:rel/)?apache-brooklyn-(\d+(?:\.\d+)+)$}i
```

The `%r{}` syntax is recommended by `brew style` when a `/` is present in the regex because using the `%r{}` syntax allows us to use forward slashes without having to escape them. As such, we can unescape the forward slash here now that the syntax is updated.

Some of the (early?) tags don't have the leading `rel/` text, so we may want to make it optional here in case they move away from this scheme or forget to add it to a release. We're restricting matching to stable versions, so we don't have to worry about picking up unstable versions if they end up publishing pre-release tags without the preceding `rel/`.

The hyphen before the numeric version seems to always be present in the tags, so it's probably safe to drop the `?` in `apache-brooklyn-?`.